### ISSUE TITLE
Incorrect kubectl binary in arm64 image

### ISSUE THREAD

**Issue by ondrejmo** [flant/shell-operator#311]
**Expected behavior (what you expected to happen)**:

`flant/shell-operator:v1.0.2` image with arm64 platform should contain `kubectl` binary compiled for arm64.

**Actual behavior (what actually happened)**:

The aforementioned image contains amd64 binary and therefore doesn't work.

**Steps to reproduce**:

```
user@arm64-system:~ $ sudo docker run -it --rm --entrypoint sh  --platform arm64 flant/shell-operator:v1.0.2 -c 'apk add file > /dev/null; file /bin/kubectl'
/bin/kubectl: ELF 64-bit LSB executable, x86-64, version 1 (SYSV), statically linked, stripped
```

**Environment**:
- Shell-operator version: 1.0.0, 1.0.1, 1.0.2
- Kubernetes version: 1.21 (doesn't really matter)
- Installation type (kubectl apply, helm chart, etc.): kubectl apply (also doesn't matter)

**Anything else we should know?**:

I believe that the cause is a missing declaration of argument (`ARG TARGETPLATFORM`) in the `Dockerfile`. Since docker buildkit docs state the following: _"These arguments are defined in the global scope so are not automatically available inside build stages or for your RUN commands. To expose one of these arguments inside the build stage redefine it without value."_ [source](https://docs.docker.com/engine/reference/builder/#automatic-platform-args-in-the-global-scope)

**Additional information for debugging (if necessary)**:

<details><summary>Hook script</summary><br><pre>

</pre></details>

<details><summary>Logs</summary><br><pre>

</pre></details>

**Comment by diafour** [comment]
Thanks for the report! I'll fix it and make a new release 1.0.3.

**Comment by diafour** [comment]
Fixed in #312
